frontend/ui/widget/bookmapwidget.lua

```diff
@@ -1357,17 +1364,24 @@ Under the pages, these indicators may be shown:
 
     if self.overview_mode then
         text = text .. "\n\n" .. _([[
-In overview mode, the book map is always in grid mode and made to fit on a single screen. Chapter levels can be changed for the most comfortable overview.]])
+When in overview mode, the book map is always displayed in grid mode to fit on one screen. The chapter levels can be easily adjusted for the most convenient overview experience.]])
     else
         text = text .. "\n\n" .. _([[
-On a newly opened book, the book map will start in grid mode showing all chapter levels, fitting on a single screen, to give the best initial overview of the book's content.]])
+When you first open a book, the book map will begin in grid mode, displaying all chapter levels on one screen for a comprehensive overview of the book's content.]])
     end
     UIManager:show(InfoMessage:new{ text = text })
 end
 
 function BookMapWidget:showGestures()
     local text
-    if self.overview_mode then
+    if not Device:isTouchDevice() then
+        text = _([[
+Use settings in this menu to change the level of chapters to include in the book map, the view type (grid or flat) and the width of page slots.
```

Comment:
"page slots" ! sounds good!

Reply:
Not convinced Page-bar is better than "page slot", but there is another "page slot" instance in the onGesture text.

Reply:
I agree that other sounds better, but you have to think from the user's perspective (that has never used this), what would make more sense to you, a page bar or a page slot?

whatever you want to go with, I'll use so say the word.

Reply:
What concretely is a "page bar?

Reply:
I know, it's a made up term. a bar that represents a page

Reply:
Okay, making me check myself… A "page bar" seems to be what I thought it might be based on what I figured "page slot" was, meaning imho page slot is much more meaningful.